decidim-core/app/helpers/decidim/user_profile_helper.rb

```diff
@@ -14,9 +14,13 @@ module UserProfileHelper
     #
     # Returns a String with the menu tab.
     def user_profile_tab(text, link, options = {})
-      active = is_active_link?(link, (options[:aria_link_type] || :inclusive))
+      aria = {}
```

Comment:
Note that there is a mistake in the original PR (#12697) since this is not used in 0.28.

The `cls` variable needs to be defined and it has to have the `tabs-title` class that used to be there.
```suggestion
      cls = %w(tabs-title)
      aria = {}
```

Reply:
Applied it in #12729 
I'm working locally in a PR to remove this helper from `develop`. Mind that as it's a `type: removal` we will not backport it. 
I'll accept this suggestion even though it's a fix in dead code. It doesn't hurt for sure.